src/components/TooltipTableCellHeader/RefTableCellHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import TableCellHeader, { TableCellHeaderProps } from "../TableCellHeader";
+
+// Workaround because child of <Tooltip /> must be able to hold ref
```

Comment:
Why?

Reply:
This is just how MUI Tooltip works - in order to show, its child has to be able to access DOM node and it is done by giving a ref to the component. Custom function components like `TableCellHeader` can't hold refs so `React.forwardRef` is needed. Further reading: https://v4.mui.com/guides/composition/#caveat-with-refs

Reply:
Directly modify TableCellHeader component and wrap it inside forwardRef instead